### PR TITLE
[ADD] stock_connector_yellowcube

### PR DESCRIPTION
**Pull request by ghost**
Added automatisation through action rules

### CODE REVIEW

File: stock_connector/models/event_processor.py [hunk 1]
```diff
@@ -6,9 +6,14 @@
 #
 #    See LICENSE file for full licensing details.
 ##############################################################################
+from openerp import api
 from openerp.addons.connector.connector import ConnectorUnit
 from .backend_processor import stock_backend_alpha
+from .constants import SQL_RELEASE, SQL_ROLLBACK
```
Comment (BT-cserra) [inline stock_connector/models/event_processor.py]: Isn't this import redundant having already the import of the following line?

File: stock_connector/models/stock_connector_backend.py [hunk 2]
```diff
@@ -10,6 +10,9 @@
 from openerp.addons.connector.session import ConnectorSession
 from openerp.addons.connector.connector \
     import ConnectorUnit, ConnectorEnvironment
+from .constants import SQL_RELEASE, SQL_ROLLBACK
```
Comment (BT-cserra) [inline stock_connector/models/stock_connector_backend.py]: Also, maybe a redundant import.

File: stock_connector/models/stock_connector_backend.py [hunk 3]
```diff
@@ -47,6 +50,38 @@ def select_versions(self):
...
+            ('name', 'in', [
+                'stock_connector.backend',
+                'stock_connector.event',
+                'stock_connector.file',
+            ])]
+        )
+        self.action_rule_ids = self.env['base.action.rule'].search([
+            ('model_id', 'in', models.ids),
+            '|',
+            ('active', '=', True),
```
Comment (BT-cserra) [inline stock_connector/models/stock_connector_backend.py:78]: ('active', 'in', [True, False]) if you prefer it.

**Review by BT-cserra — CHANGES_REQUESTED** [review]

File: stock_connector/models/stock_connector_backend.py (hunk 3 above)
Comment (ghost) [reply stock_connector/models/stock_connector_backend.py:78]: For some reason it doesnot work with that

**Pull request merged**